mbed_tools_ci_scripts/spdx_report/spdx_package.py

```diff
@@ -206,7 +223,7 @@ def generate_spdx_package(self) -> Package:
         package.download_location = determine_spdx_value(None)
         package.homepage = determine_spdx_value(self.url)
         package.originator = Person(determine_spdx_value(self.author), determine_spdx_value(self.author_email))
-        package.license_declared = License.from_identifier(str(determine_spdx_value(self.main_licence)))
+        package.license_declared = License.from_identifier(str(determine_spdx_value(self.main_licence.identifier)))
```

Comment:
Would it make more sense to set these attributes in `Package.__init__` rather than monkey patching? We could just pass a reference to `self` to the `Package` and then set the attributes when we construct `Package`. It would make this calling code easier to follow.

Reply:
I did what I could. Package is a class provided by the SDK and the __init__ does not have all the attributes defined. Only set the one I could

Reply:
418326abd2b04ddfb63cea5c08fe58e7951442d4